octoprint_mrbeam/templates/mrbeam_ui_index.jinja2

```diff
@@ -51,9 +51,9 @@
 					<li class="active"><a href="#workingarea" data-toggle="tab" id="wa_tab_btn">{{ _('working area') }}</a></li>
                     {# Translatots: Name of the tab #}
 					<li><a href="#designlib" data-toggle="tab" id="designlib_tab_btn">{{ _('design library') }}</a></li>
-                    <li><a href="#designstore" data-toggle="tab" id="designstore_tab_btn" onclick="window.mrbeam.viewModels.designStoreViewModel.goToStore()">{{ _('design store') }}</a></li>
+                    <li><a href="#designstore" data-toggle="tab" id="designstore_tab_btn" onclick="window.mrbeam.viewModels.materialStoreViewModel.initialiseStore()">{{ _('design store') }}</a></li>
```

Comment:
This seems wrong.. we are calling initilaliseStore() from materialStoreViewModel... We are also not adding the lazy loading tag on the iframe.... Is this code review ready?

Reply:
no it wasn't ready